templates/rustdoc/topbar.html

```diff
@@ -222,9 +222,14 @@
                     {%- set target_url = "/crate/" ~ metadata.name ~ "/" ~ metadata.version_or_latest ~ "/target-redirect/" ~ target ~ "/" ~ inner_path -%}
                     {%- set target_no_follow = "nofollow" -%}
                 {%- endif -%}
+                {%- if current_target is defined and current_target == target -%}
```

Comment:
I did it this way because if it's not a "rustdoc page" (like crate docs.rs page, or features, etc), then the platform isn't actually used (in the URL) so no need to set `current_target`. But since we still render the topbar, we need to check if `current_target` is set before actually using it.